portfolio/src/main/webapp/art.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Federick's Art</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/favicon.ico"/>
+    <link href="https://fonts.googleapis.com/css2?family=Nanum+Myeongjo:wght@700&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="header">
+        <a href="index.html" class="logo">Federick Gonzalez</a>
+        <div class="header-right">
+            <a  href="index.html">Home</a>
+            <a href="profiles.html">Art</a>
+            <div class="dropdown">
+                <button class="dropbtn">More
+                    <i class="fa fa-caret-down"></i>
+                </button>
+                <div class="dropdown-content">
+                    <a href="photography.html">Photos</a>
+                     <a href="profiles.html">Profiles</a>
+                    <a href="martialArt.html">MMA</a>
+                </div> 
+            </div>
+        </div>
+    </div>
+     <div class="bg-text">
+        <h1>Art</h1>
+        <p>Though photography has interested me, I probably would prefer creating my own art as opposed to photographing moments around me. Below are a selection of a few of the pieces I've made. There are several more, though a lot do not have photos, and others are in a notebook that I cannot find. I hope these are enjoyable, and I also hope to be creating more soon!</p>
```

Comment:
Split into multiple lines here like

<p>
    ....
    ....
</p>

Reply:
Broke down all the longer lines into multiple lines in the submitted files